src/main/resources/static/vulnerableApp.js

```diff
@@ -69,8 +80,8 @@ function _callbackForInnerMasterOnClickEvent(
       dynamicScriptNode = parentNodeWithAllDynamicScripts.lastElementChild;
     }
     doGetAjaxCall((responseText) => {
-      detailTitle.innerHTML = responseText;
```

Comment:
CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')
------
User controlled data in methods like `innerHTML`, `outerHTML` or `document.write` is an anti-pattern that can lead to XSS vulnerabilities